specification.txt

```diff
@@ -2373,9 +2373,11 @@ items is performed by:
    is capitalized. The case of words in mixed or uppercase stays the same.
    
 In both cases, stop words are lowercased, unless they are the first or last word
-in the string, or follow a colon. The stop words are "a", "an", "and", "as",
-"at", "but", "by", "down", "for", "from", "in", "into", "nor", "of", "on",
-"onto", "or", "over", "so", "the", "till", "to", "up", "via", "with", and "yet".
+in the string, or follow a colon. The stop words are listed in the file
+`stop-words.json
+<https://raw.github.com/citation-style-language/documentation/master/stop-words.json>`_.
```

Comment:
Per https://github.com/citation-style-language/schema/issues/216#issuecomment-638415275, this should probably become something like https://resource.citationstyles.org/.../stop-words.json.

Reply:
We want the json file in the schema repo; right?

I put it under `/schemas/styles/stop-words.json`.

Is that what you were thinking?

If yes, need to get the redirect to work.

Reply:
Or did you want to keep it here?

Reply:
We should serve the file stored at `/schemas/styles/stop-words.json` out of a link from `https://resource.citationstyles.org/`

Reply:
> We should serve the file stored at `/schemas/styles/stop-words.json` out of a link from `https://resource.citationstyles.org/`

@dstillman - could you please set up the redirect so we can point to the styles schema files; for example, this one?

https://github.com/citation-style-language/schema/blob/master/schemas/styles/stop-words.json

Reply:
So, that would specifically be any URL with the pattern `https://resource.citationstyles.org/schemas/(.*)` redirects to `https://github.com/citation-style-language/schema/blob/master/schemas/(.*)` ?

Reply:
We're currently redirecting `schema/latest/input/json/` to `citation-style-language/schema/master/schemas/input/`.

Should that remain, or should it switch solely to `schemas` (plural) URLs?

Reply:
> We're currently redirecting `schema/latest/input/json/` to `citation-style-language/schema/master/schemas/input/`.
> 
> Should that remain, or should it switch solely to `schemas` (plural) URLs?

The current should remain; we just need to also be able to access the `styles` sibling directory.

Reply:
I just need the exact path to redirect.

Reply:
You can't use @bwiernik's suggestion to redirect the common parent directory; 

`https://resource.citationstyles.org/schemas/(.*)` to 
`https:///raw.githubusercontent.com/citation-style-language/schema//master/schemas/(.*)`?

If not, would be this specific file redirect:

`https://resource.citationstyles.org/schemas/styles/stop-words.json` to
`https://raw.githubusercontent.com/citation-style-language/schema/master/schemas/styles/stop-words.json`

Reply:
Better: `https://resource.citationstyles.org/schema/latest/(.*)` to
`https://raw.githubusercontent.com/citation-style-language/schema/master/schemas/(.*)`

Reply:
OK, redirects in place:

  * `schema/latest/input/json/` → `citation-style-language/schema/master/schemas/input/`
    * Example: https://resource.citationstyles.org/schema/latest/input/json/csl-citation.json
  * `schema/latest/` → `citation-style-language/schema/master/schemas/`
    * Example: https://resource.citationstyles.org/schema/latest/styles/stop-words.json